test/cpp/qps/driver.cc

```diff
@@ -363,8 +363,6 @@ std::unique_ptr<ScenarioResult> RunScenario(
   if (run_inproc) {
     g_inproc_servers = new std::vector<grpc::testing::Server*>;
   }
-  // Log everything from the driver
-  gpr_set_log_verbosity(GPR_LOG_SEVERITY_DEBUG);
 
```

Comment:
This is really not a good place to add a debug log. This kind of work should not be done in the middle of some test code. 
This should ideally be done in the main() function of the test, or setup or some other place.